src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -1099,88 +1097,55 @@ protected override void Dispose(bool disposing)
                 // around the same OS handle, so closing one handle would invalidate
                 // them all.  Additionally, we want a second AppDomain to be able to
                 // write to stdout if a second AppDomain quits.
-                _handle = IntPtr.Zero;
+                _handle = 0;
                 base.Dispose(disposing);
             }
 
-            public override int Read(Span<byte> buffer)
-            {
-                int errCode = ReadFileNative(_handle, buffer, _isPipe, out int bytesRead, _useFileAPIs);
-                if (Interop.Errors.ERROR_SUCCESS != errCode)
-                {
-                    throw Win32Marshal.GetExceptionForWin32Error(errCode);
-                }
-
-                return bytesRead;
-            }
-
-            public override void Write(ReadOnlySpan<byte> buffer)
-            {
-                int errCode = WriteFileNative(_handle, buffer, _useFileAPIs);
-                if (Interop.Errors.ERROR_SUCCESS != errCode)
-                {
-                    throw Win32Marshal.GetExceptionForWin32Error(errCode);
-                }
-            }
-
             public override void Flush()
             {
-                if (_handle == IntPtr.Zero) throw Error.GetFileNotOpen();
-                base.Flush();
```

Comment:
Why is it safe to remove the call to `base.Flush` here? Is it a `nop`?

Reply:
Yes, it's a nop.